package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-test-utils-helpers",
-  "version": "1.0.3",
+  "version": "1.0.5",
```

Comment:
Why are we skipping `1.0.4`?

Reply:
I missed updating package.json with correct version last time (1.0.4). 

we already have a release 1.0.4 but package.json is still version. this pr syncs release and package.json